package.json

```diff
@@ -17,7 +17,9 @@
     "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
     "test": "jest -c jest.config.ts",
     "test:integration": "jest -c tests/integration-tests/jest.integration.config.ts",
-    "test:integration:dev": "jest --setupFiles '<rootDir>/.env' -c tests/integration-tests/jest.integration.config.ts"
+    "test:integration:override": "jest --setupFiles '<rootDir>/.env' -c tests/integration-tests/jest.integration.config.ts",
+    "test:e2e": "jest -c tests/e2e-tests/jest.e2e.config.ts",
+    "test:e2e:override": "jest --setupFiles '<rootDir>/.env' -c tests/e2e-tests/jest.e2e.config.ts"
```

Comment:
Can we change this back to `test:integration:dev` + `test:e2e:dev` to keep consistency

Reply:
Andy thought that we might actually change this in the other repo, as it serves to help run against any environment. I'm happy to change it, just thought you might see it that way.